source/regex.tex

```diff
@@ -869,7 +869,7 @@
 \tcode{error_stack}
 &
 There was insufficient memory to determine whether the regular
-expression could match the specified character sequence.  \\
+expression matched the specified character sequence.  \\
```

Comment:
Using the past tense here seems wrong. If an exception was thrown, it didn't match. Maybe "matches" or "would match"?

Reply:
It's conditional, not past tense.

Reply:
But "would match" sounds good.

Reply:
fixed + force-pushed